app/controllers/users/registrations_controller.rb

```diff
@@ -1,19 +1,20 @@
 class Users::RegistrationsController < Devise::RegistrationsController
-    respond_to :json
-  
-    private
-  
-    def respond_with(resource, _opts = {})
-      register_success && return if resource.persisted?
-  
-      register_failed resource
-    end
-  
-    def register_success
-      render json: { message: 'Signed up sucessfully.' }, status: :created
-    end
-  
-    def register_failed resource
-      render json: { message: resource.errors.full_messages }, status: :bad_request
-    end
-  end
\ No newline at end of file
+  private
+
+  def respond_with(resource, _opts = {})
+    register_success && return if resource.persisted?
+
+    register_failed resource
+  end
+
+  def register_success
+    redirect_to root_path, notice: 'Signed up sucessfully.'
+  end
+
+  def register_failed(resource)
+    print(resource.errors.full_messages)
```

Comment:
Is this a debug line? Looks like `_error_messages.html.erb` shows errors to the user.

Reply:
this is what i pulled from dev-main.